<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="partInfoGroupBox">
     <property name="title">
      <string>Part Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="partList">
          <property name="text">
           <string>Part List </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="partListCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="partName">
          <property name="text">
           <string>Part Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="partNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="fileName">
          <property name="text">
           <string>File Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileNameLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseBtn">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="PAgroupBox">
     <property name="title">
      <string>Position and Attitude</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="XLabel">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="XLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="YLabel">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="YLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="ZLabel">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ZLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="Heading">
          <property name="text">
           <string>Heading</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="HLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Pitch">
          <property name="text">
           <string>Pitch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="Roll">
          <property name="text">
           <string>Roll</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ApplyBtn">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OkBtn">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
